src/features/events/components/EventsList/index.tsx

```diff
@@ -29,13 +34,9 @@ export const EventsList: FC = () => {
     if (filter === FilterType.ALL) {
       return [...data].sort(sorts.ascending)
     } else if (filter === FilterType.FUTURE) {
-      return data
-        .filter((event) => isAfter(new Date(event.startsAt), new Date()))
-        .sort(sorts.ascending)
+      return data.filter(filters.future).sort(sorts.ascending)
```

Comment:
Very simple to read. If there is a filtering bug, we only have to look at the `filters.future` function, not on the component body code anymore.